DataManagementSystem/scripts/dirac-dms-show-se-status.py

```diff
@@ -77,9 +77,12 @@ def setNoVO( arg ):
 for se, statusDict in res[ 'Value' ].items():
 
   # Check if the SE is allowed for the user VO
-  voList = gConfig.getValue( '/Resources/StorageElements/%s/VO' % se, [] )
-  if voList and not vo in voList:
-    continue
+  if not allVOsFlag:
+    voList = gConfig.getValue( '/Resources/StorageElements/%s/VO' % se, [] )
+    if noVOFlag and voList:
+      continue
+    if voList and not vo in voList:
```

Comment:
Basically that's not part of my code, I saw what is the current file here: https://github.com/DIRACGrid/DIRAC/blob/rel-v6r15/DataManagementSystem/scripts/dirac-dms-show-se-status.py#L80 and copied it to fix a conflict. 

Should I change it?